### PR TITLE
feat(cli): new agent access token command

### PR DESCRIPTION
**Pull request by afiune**
### New `lacework agent` command!
```
$ lacework agent --help
Manage agents and agent access tokens in your account.

To analyze application, host and user behavior, Lacework uses a lightweight agent,
which securely forwards collected metadata to the Lacework cloud for analysis. The
agent requires minimal system resources and runs on most 64-bit Linux distributions.

For a complete list of supported operating systems, visit:

    https://support.lacework.com/hc/en-us/articles/360005230014-Supported-Operating-Systems

Usage:
  lacework agent [command]

Available Commands:
  token       manage agent access tokens

Flags:
  -h, --help   help for agent

Global Flags:
  -a, --account string      account subdomain of URL (i.e. <ACCOUNT>.lacework.net)
  -k, --api_key string      access key id
  -s, --api_secret string   secret access key
      --debug               turn on debug logging
      --json                switch commands output from human-readable to json format
      --nocolor             turn off colors
      --noninteractive      turn off interactive mode (disable spinners, prompts, etc.)
  -p, --profile string      switch between profiles configured at ~/.lacework.toml

Use "lacework agent [command] --help" for more information about a command.
```

This command contains only one sub-command at the moment, the `lacework agent token` command to manage agent access tokens. In the future, we are planning to extend this agent command to do other things like list agents, generate deployment scripts, and perhaps even install agents on remote hosts.

tenor-126201367

JIRA: https://lacework.atlassian.net/browse/ALLY-236

Contributes to https://github.com/lacework/terraform-provider-lacework/issues/41

Signed-off-by: Salim Afiune Maya <afiune@lacework.net>

### CODE REVIEW

File: integration/agent_token_test.go [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+		assert.Equal(t, 0, exitcode,
+			"EXITCODE is not the expected one")
+
+		var token agentToken
+		errJson := json.Unmarshal(out.Bytes(), &token)
+		assert.Nil(t, errJson)
+		assert.NotEmpty(t, token, "check JSON token show response while parsing update time")
+		assert.True(t,
+			tokenUpdatedTime.Before(token.LastUpdatedTime),
+			fmt.Sprintf("the agen access token last_update_time was NOT updated! check please. (old:%s) (new:%s)",
```
Comment (scottford-lw) [inline integration/agent_token_test.go:200]:
```suggestion
			fmt.Sprintf("the agent access token last_update_time was NOT updated! check please. (old:%s) (new:%s)",
```

File: cli/cmd/agent.go [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+	tokenUpdateDesc    string
+
+	agentCmd = &cobra.Command{
+		Use:   "agent",
+		Short: "manage Lacework agents",
+		Long: `Manage agents and agent access tokens in your account.
+
+To analyze application, host and user behavior, Lacework uses a lightweight agent,
+which securely forwards collected metadata to the Lacework cloud for analysis. The
+agent requires minimal system resources and runs on most 64-bit Linux distributions.
```
Comment (scottford-lw) [inline cli/cmd/agent.go:47]:
```suggestion
To analyze application, host, and user behavior, Lacework uses a lightweight agent,
which securely forwards collected metadata to the Lacework cloud for analysis. The
agent requires minimal system resources and runs on most 64-bit Linux distributions.
```

Comment (scottford-lw) [reply cli/cmd/agent.go:47]: @afiune Did you pull this text from support docs, or was it created new for the CLI?

**Review by scottford-lw — APPROVED** [review]
@afiune Great stuff. I had a couple of minor finds, but 👍

File: cli/cmd/agent.go (hunk 2 above)
Comment (afiune) [reply cli/cmd/agent.go:47]: From https://support.lacework.com/hc/en-us/articles/360005230014-Supported-Operating-Systems

**Pull request merged**